discli

```diff
@@ -16,20 +17,33 @@ from urllib.parse import urljoin
 
 dotfile = ".discli"
 
+def counter(start=0):
+    i = start
+    while True:
+        yield i
+        i = i + 1
+
+def now():
+    return datetime.datetime.utcnow().isoformat(sep='T', timespec='minutes') + '+00:00'
 
 class DiscourseHelper:
 
     def __init__(self):
+        self.http = requests.Session()
         parser = argparse.ArgumentParser(
             description='Helper to work with discourse files locally',
-            usage=textwrap.dedent('''
-                discli <command> [<args>]
-
-                The commands are:
-                   init      Record your api key and target discourse
-                   get       Get the file
-                   put       Put a file back
-                   get_posts_in_category  Get all the posts in the given category name
+            usage=textwrap.dedent('''\
+                discli <command> [<arg> ...]
+
+                Commands:
+                   init <url> <username> <api_key>   Record your API key and target Discourse
+                   forget                            Deletes your credentials
+                
+                Commands that interact with Discourse directly:
+                   get <topic-id>                    Download a Discourse topic 
+                   copy <topic-id>                   Update a topic with the local file's contents
```

Comment:
"update" ?

Reply:
Also, instead of two verbs, just a --keep ?

Reply:
Good ideas